pulsar-broker/src/test/java/org/apache/pulsar/client/api/SimpleSchemaTest.java

```diff
@@ -1233,6 +1233,79 @@ public void testAutoCreatedSchema(String domain) throws Exception {
         Assert.assertEquals(admin.schemas().getSchemaInfo(topic2).getType(), SchemaType.STRING);
     }
 
+    @Test(dataProvider = "topicDomain")
+    public void testSubscribeWithSchemaAfterAutoConsumeNewTopic(String domain) throws Exception {
```

Comment:
Why did you create 4 consumers for each schema type? It's unnecessary and we only need 1 consumer for each schema type.

Reply:
Previous old design I needed to test the case of multiple consumers with different subscriptions to subscribe a topic.

Reply:
What do you think of such test cases? I think that's a little more "detailed"?

Reply:
It makes sense to me.